lib/kinetic.js

```diff
@@ -1615,6 +1615,50 @@ class EndBatchResponsePDU extends PDU {
     }
 }
 
+/**
+ * Get error from parsed PDU
+ * @param {Object} pdu - parse pdu
+ * @returns {String | undefined} error message
+ */
+function getPduError(pdu) {
```

Comment:
I don't understand this function you can use : 
https://github.com/lamphamsy/kinetic-js/blob/9aa693330ce08efe032cfd511afdbb3a0bfa3d16/lib/kinetic.js#L440
and
https://github.com/lamphamsy/kinetic-js/blob/9aa693330ce08efe032cfd511afdbb3a0bfa3d16/lib/kinetic.js#L146

you can modify them if it's needed